hacs.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "HACS",
   "zip_release": true,
-  "hide_default_branch": true,
+  "hide_default_branch": false,
```

Comment:
HACS relies on translation files that are downloaded and placed in the correct directory when building the release asset.
Due to that, allowing master installation will not happen.